spec/helpers/service_helper/textual_summary_spec.rb

```diff
@@ -0,0 +1,30 @@
+describe ServiceHelper::TextualSummary do
+  context ".textual_orchestration_stack" do
+    before do
+      login_as @user = FactoryGirl.create(:user)
+    end
+
+    subject { textual_orchestration_stack }
+    it 'contains the link to the associated stack' do
+      @os_cloud = FactoryGirl.create(:orchestration_stack_cloud, :name => "cloudstack1")
+      @record = FactoryGirl.create(:service)
+      allow(@record).to receive(:orchestration_stack).and_return(@os_cloud)
+      expect(textual_orchestration_stack).to eq(@os_cloud)
+    end
+
+    it 'contains the link to the associated stack' do
```

Comment:
the description is the same for both cases, can you add cloud & infra to the descriptions so that they differ please?